myFirstWebsite/index.html

```diff
@@ -0,0 +1,445 @@
+                            <div class="top-places__item-ui">
+                                <h3 class="top-places__item-title">
+                                    UENO
+                                </h3>
+                                <div class="top-places__item-likes">
+                                    <span class="likes-amount">228</span>
+                                    <img src="./images/like-icon.svg" alt="like" class="like-icon">
+                                </div>
+                                <div class="top-places__item-comments">
+                                    <span class="comments-amount">228</span>
+                                    <img src="./images/comment-icon.svg" alt="comment" class="comment-icon">
+                                </div>
+                            </div>
+                        </div>
+                        <p class="top-places__item-description">
+                            Jump off balcony, onto stranger's head. Chase ball of string hide when guests come over. Being gorgeous with belly side up i could pee on this.
+                        </p>
+                        <button type="button" class="top-places__more-btn common-btn">see more</button>
+                    </div>
+                    <div class="top-places__item">
+                        <div class="top-places__item-image-holder">
+                            <img src="../myFirstWebsite/images/top-places-6.jpg" alt="SHIBUYA" class="top-places__item-image">
+                            <div class="top-places__item-ui">
+                                <h3 class="top-places__item-title">
+                                    SHIBUYA
+                                </h3>
+                                <div class="top-places__item-likes">
+                                    <span class="likes-amount">228</span>
+                                    <img src="./images/like-icon.svg" alt="like" class="like-icon">
+                                </div>
+                                <div class="top-places__item-comments">
+                                    <span class="comments-amount">228</span>
+                                    <img src="./images/comment-icon.svg" alt="comment" class="comment-icon">
+                                </div>
+                            </div>
+                        </div>
+                        <p class="top-places__item-description">
+                            Jump off balcony, onto stranger's head. Chase ball of string hide when guests come over. Being gorgeous with belly side up i could pee on this.
+                        </p>
+                        <button type="button" class="top-places__more-btn common-btn">see more</button>
+                    </div>
+                </div>
+            </div>
+        </section>
+        <div class="videos" id="videos">
+            <div class="videos__wrapper">
+                <div class="videos__adaptive-wrapper">
```

Comment:
Very nice idea! Honestly I didn't find a proper solution to scale up a smaller video, but you managed it nicely! Hovewer, you don't need 2 wrappers here, you can have 1 like this: 


```
.videos__adaptive-wrapper{

    max-width: 1200px;
    margin: 0 auto;
    position: relative;
    padding-top: 30px;
    aspect-ratio: 16 / 9;
    overflow: hidden;
} 
```

Reply:
+